### DISCUSSION TITLE
a-bootstrapper-script-for-the-cake-net-core-global-tool-on-windows/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#68] [Announcements]
# a-bootstrapper-script-for-the-cake-net-core-global-tool-on-windows/

In this post I provide a Windows PowerShell bootstrapper for the Cake.Tool .NET Core global tool.

https://andrewlock.net/a-bootstrapper-script-for-the-cake-net-core-global-tool-on-windows/

[a comment by an automated account is not shown]

**Reply by andrewlock** [reply]
<em>[Sock](https://disqus.com/by/andrewlockdotnet/) (@andrewlock) commented [on Disqus](https://disqus.com/home/forum/andrewlock/) at August 21 2019, 10:39, in reply to [James Webster](https://disqus.com/by/disqus_66iKFb5XQG/)'s [comment](https://github.com/andrewlock/blog-comments/discussions/68#discussioncomment-6057669)</em>

---

<p>🤞!</p>
